Cargo.toml

```diff
@@ -118,33 +118,34 @@ substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", bran
 try-runtime-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
 
 # Polkadot (wasm)
-polkadot-parachain = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.40", default-features = false }
-polkadot-runtime-common = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.40", default-features = false }
+polkadot-parachain = { git = "https://github.com/tmpolaczyk/polkadot", branch = "tomasz-reinitialize-9-40", default-features = false }
+polkadot-runtime-common = { git = "https://github.com/tmpolaczyk/polkadot", branch = "tomasz-reinitialize-9-40", default-features = false }
```

Comment:
Can you use our fork? https://github.com/moondance-labs/polkadot. You can put polkadot-v0.9.40 as the branch or tanssi-polkadot-v0.9.40